index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+  <html lang="en">
+    <head>
+      <meta charset="utf-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>My Website Portfolio</title>
+      <link rel="stylesheet" href="main.css">
+      <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+    </head>
+    <body>
+      <!-- This div holds the entire content of the main page to add a common background for it in the styling file -->
+      <div class="main-page-background">
+        <header class="main-page-header">
+          <h1 class="logo">Karla Delgado</h1>
```

Comment:
- [ ] kindly try to wrap your logo inside an `a` tag so that it is even clickable .doing so will make your project more dynamic and interactive .